<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="answerBtn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ОТВЕТ 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerBtn2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerBtn3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerBtn4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiftyFiftyBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audienceBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="callBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeMoneyBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMenuBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>151</width>
      <height>381</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1000000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>700000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>400000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1. 100</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="questionLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>531</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="valueLevelLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вопрос 1/15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
